codes.md

```diff
@@ -311,7 +304,7 @@ These codes describe observations about mental models and their formation.
     - [Session 022](https://osf.io/psv9g/) - basket drawn for carrying
 
 
-### MM6. Physical objects can also be used to represent data
+### MM6. Physical objects can be used to represent data.
   - This code cross-references mental models, depictions, AND what-is-data,
```

Comment:
Maybe "this code cross-references the themes mental models, depictions, AND what-is-data..."